sdk/digitaltwins/Azure.DigitalTwins.Core/tests/PublishTelemetryTests.cs

```diff
@@ -47,26 +47,18 @@ public async Task PublishTelemetry_Lifecycle()
                 await CreateModelsAndTwins(client, wifiModelId, roomWithWifiModelId, wifiComponentName, roomWithWifiTwinId).ConfigureAwait(false);
 
                 // Act - Test publishing telemetry to a digital twin.
-                var telemetryOptions = new PublishTelemetryOptions()
-                {
-                    TimeStamp = default
-                };
-                Response publishTelemetryResponse = await client.PublishTelemetryAsync(roomWithWifiTwinId, Recording.Random.NewGuid().ToString(), "{\"Telemetry1\": 5}", telemetryOptions).ConfigureAwait(false);
+                Response publishTelemetryResponse = await client.PublishTelemetryAsync(roomWithWifiTwinId, Recording.Random.NewGuid().ToString(), "{\"Telemetry1\": 5}", timeStamp: default(DateTimeOffset)).ConfigureAwait(false);
```

Comment:
nit: line seems too long. we can break it into multiple

Reply:
Split across lines